### PR TITLE
MM-46990 Report correct import error

### PR DESCRIPTION
**Pull request by noxer**
#### Summary
The import reports all errors as `We could not count the users.` and since the last error improvements it doesn't report the original error anymore.

#### Ticket Link
[MM-46990](https://mattermost.atlassian.net/browse/MM-46990)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by agnivade** [comment]
Ahem .. should I close https://github.com/mattermost/mattermost-server/pull/21030 then ? :P

**Comment by noxer** [comment]
@agnivade first a question. Why did you add the "no translation" key there?

**Comment by agnivade** [comment]
They are just logged in the DB right? Not sent the the webapp. So there isn't a need to translate them.

**Comment by noxer** [comment]
They are logged into the DB and then retrieved by `mmctl` to display the state of the job.

**Comment by agnivade** [comment]
It's okay to keep those non-translated.

File: i18n/en.json [hunk 1]
```diff
@@ -5607,6 +5607,10 @@
     "id": "app.job.download_export_results_not_enabled",
     "translation": "DownloadExportResults in config.json is false. Please set this to true to download the results of this job."
   },
+  {
+    "id": "app.job.error",
+    "translation": "Error during job execution."
```
Comment (cwarnermm) [inline i18n/en.json:5612]: Is there anything the user who sees this message can do as a workaround? If so, do we want to add those workaround details to the error message?

**Review by isacikgoz — APPROVED** [review]

File: i18n/en.json (hunk 1 above)
Comment (noxer) [reply i18n/en.json:5612]: This is just a generic message for when a Job fails. What the job is doing exactly is unknown at this point. The error message presented to the user (for example the admin importing a backup into mattermost) consists of this message and the error reported by the actual job.

**Comment by noxer** [comment]
@agnivade Due to how the import job is designed, that would print the "no translation" key to the end user, I'd prefer to show at least a generic message.

**Comment by agnivade** [comment]
@noxer - Ah yes, I think we would need to change the `(er *AppError) Error()` method as well to ignore model.NoTranslation keys. Anyways, I'll close off my PR then.

**Review by agnivade — APPROVED** [review]

**Pull request merged**